data/strings/strings.txt

```diff
@@ -22978,49 +22978,93 @@
 
   [export_file]
     tags = android,ios
-    en = Export file
-    af = Stuur lêer uit
-    ar = ملف التصدير
-    az = Faylı ixrac edin
-    be = Экспартаваць файл
-    bg = Експортиране на файл
-    ca = Exporta com a fitxer
-    cs = Exportovat soubor
-    da = Eksporter fil
-    de = Datei exportieren
-    el = Εξαγωγή Αρχείου
-    es = Exportar archivo
-    es-MX = Exportar archivo
-    et = Ekspordi fail
-    eu = Esportatu fitxategia
-    fa = ارسال فایل
-    fi = Vie tiedosto
-    fr = Exporter le fichier
-    he = יצא קובץ
-    hi = फ़ाइल निर्यात करें
-    hu = Fájl exportálása
-    id = File ekspor
-    it = Esportare il file
-    ja = ファイルのエクスポート
-    ko = 파일 내보내기
-    lt = Eksporuoti failą
-    mr = फाइल निर्यात
-    nb = Eksporter fil
-    nl = Bestand exporteren
-    pl = Eksportuj plik
-    pt = Exportar o ficheiro
-    pt-BR = Exportar arquivo
-    ro = Exportă fișierul
-    ru = Экспортировать файл
-    sk = Exportovať súbor
-    sv = Exportera fil
-    sw = Tuma faili
-    th = ส่งออกไฟล์
-    tr = Dosyayı dışa aktar
-    uk = Експортувати файл
-    vi = Xuất tập tin
-    zh-Hans = 导出文件
-    zh-Hant = 導出文件
+    en = Export KML/KMZ file
+    af = Voer KML/KMZ-lêer uit
+    ar = تصدير ملف KML/KMZ
+    az = KML/KMZ faylını ixrac edin
+    be = Экспарт файла KML/KMZ
+    bg = Експортиране на KML/KMZ файл
+    ca = Exporta el fitxer KML/KMZ
+    cs = Export souboru KML/KMZ
+    da = Eksporter KML/KMZ-fil
+    de = KML/KMZ-Datei exportieren
+    el = Εξαγωγή αρχείου KML/KMZ
+    es = Exportar archivo KML/KMZ
+    et = KML/KMZ-faili eksportimine
+    eu = Esportatu KML/KMZ fitxategia
+    fa = فایل KML/KMZ را صادر کنید
+    fi = Vie KML/KMZ-tiedosto
+    fr = Exporter un fichier KML/KMZ
+    he = ייצא קובץ KML/KMZ
+    hi = KML/KMZ फ़ाइल निर्यात करें
+    hu = KML/KMZ fájl exportálása
+    id = Ekspor file KML / KMZ
+    it = Esportazione di file KML/KMZ
+    ja = KML/KMZファイルをエクスポートする
+    ko = KML/KMZ 파일 내보내기
+    lt = Eksportuoti KML/KMZ failą
+    mr = KML/KMZ फाइल निर्यात करा
+    nb = Eksporter KML/KMZ-fil
+    nl = KML/KMZ-bestand exporteren
+    pl = Eksportuj plik KML/KMZ
+    pt = Exportar ficheiro KML/KMZ
+    pt-BR = Exportar arquivo KML/KMZ
+    ro = Exportați fișierul KML/KMZ
+    ru = Экспорт файла KML/KMZ
+    sk = Exportovať súbor KML/KMZ
+    sv = Exportera KML/KMZ-fil
+    sw = Hamisha faili ya KML/KMZ
+    th = ส่งออกไฟล์ KML/KMZ
+    tr = KML/KMZ dosyasını dışa aktar
+    uk = Експорт KML/KMZ файлу
+    vi = Xuất tệp KML/KMZ
+    zh-Hans = 导出 KML/KMZ 文件
+    zh-Hant = 匯出 KML/KMZ 文件
+
+  [export_file_gpx]
+    tags = android,ios
+    en = Export GPX file
```

Comment:
Let's try to make it shorter, it looks too verbose now:
```
    en = Export GPX
    af = Voer GPX uit
    ar = تصدير GPX
    az = GPX ixrac edin
    be = Экспарт GPX
    bg = Експортиране на GPX
    ca = Exporta GPX
    cs = Export GPX
    da = Eksporter GPX
    de = GPX exportieren
    el = Εξαγωγή GPX
    es = Exportar GPX
    et = Ekspordi GPX
    eu = Esportatu GPX
    fa = صادرات GPX
    fi = Vie GPX
    fr = Exporter GPX
    he = ייצוא GPX
    hi = जीपीएक्स निर्यात करें
    hu = GPX exportálása
    id = Ekspor GPX
    it = Esportazione GPX
    ja = GPXをエクスポートする
    ko = GPX 내보내기
    lt = GPX eksportavimas
    mr = GPX निर्यात करा
    nb = Eksporter GPX
    nl = GPX exporteren
    pl = Eksport GPX
    pt = Exportar GPX
    ro = Exportați GPX
    ru = Экспорт GPX
    sk = Export GPX
    sv = Exportera GPX
    sw = Hamisha GPX
    th = ส่งออก GPX
    tr = GPX'i Dışa Aktar
    uk = Експорт GPX
    vi = Xuất GPX
    zh-Hans = 导出 GPX
    zh-Hant = 導出GPX
```

Reply:
```
    en = Export KMZ
    af = Voer KMZ uit
    ar = تصدير KMZ
    az = KMZ ixrac edin
    be = Экспарт KMZ
    bg = Експорт на KMZ
    ca = Exporta KMZ
    cs = Export KMZ
    da = Eksporter KMZ
    de = KMZ exportieren
    el = Εξαγωγή KMZ
    es = Exportar KMZ
    et = KMZ eksportimine
    eu = Esportatu KMZ
    fa = صادرات KMZ
    fi = Vie KMZ
    fr = Exporter KMZ
    he = ייצא KMZ
    hi = निर्यात KMZ
    hu = KMZ exportálása
    id = Ekspor KMZ
    it = Esportazione KMZ
    ja = KMZをエクスポートする
    ko = KMZ 내보내기
    lt = Eksportuoti KMZ
    mr = KMZ निर्यात करा
    nb = Eksporter KMZ
    nl = KMZ exporteren
    pl = Eksportuj KMZ
    pt = Exportar KMZ
    ro = Export KMZ
    ru = Экспорт KMZ
    sk = Exportovať KMZ
    sv = Exportera KMZ
    sw = Hamisha KMZ
    th = ส่งออก KMZ
    tr = KMZ'yi Dışa Aktar
    uk = Експорт КМЗ
    vi = Xuất KMZ
    zh-Hans = 导出 KMZ
    zh-Hant = 出口KMZ
```

Reply:
Fixed

Reply:
The code did not change, strange...